linkml_model/model/docs/specification/03schemas.md

```diff
@@ -72,22 +72,22 @@ However, it should be understood these are not precisely equivalent.
 
 A schema *m* is an instance of a SchemaDefinition, with normative elements:
 
-| Name | Cardinality and Range  | Description  |
-| ---  | ---  | --- |
-| [id](id.md) | 1..1 <br/> [xsd:anyURI](http://www.w3.org/2001/XMLSchema#anyURI)  | The official schema URI   |
-| [name](name.md) | 1..1 <br/> [xsd:string](http://www.w3.org/2001/XMLSchema#string)  | a unique name for the schema that is both human-readable and consists of only... **identifier**  |
-| [classes](classes.md) | 0..* <br/> [ClassDefinition](ClassDefinition.md)  | An index to the collection of all class definitions in the schema   |
-| [slot_definitions](slot_definitions.md) | 0..* <br/> [SlotDefinition](SlotDefinition.md)  | An index to the collection of all slot definitions in the schema   |
-| [enums](enums.md) | 0..* <br/> [EnumDefinition](EnumDefinition.md)  | An index to the collection of all enum definitions in the schema   |
-| [subsets](subsets.md) | 0..* <br/> [SubsetDefinition](SubsetDefinition.md)  | An index to the collection of all subset definitions in the schema   |
-| [prefixes](prefixes.md) | 0..* <br/> [Prefix](Prefix.md)  | prefix / URI definitions to be added to the context beyond those fetched from...   |
-| [default_prefix](default_prefix.md) | 0..1 <br/> [xsd:string](http://www.w3.org/2001/XMLSchema#string)  | default and base prefix -- used for ':' identifiers, @base and @vocab   |
-| [default_range](default_range.md) | 0..1 <br/> [TypeDefinition](TypeDefinition.md)  | default slot range to be used if range element is omitted from a slot definit...   |
```

Comment:
This is probably tangential to your changes, but what's up with all these tables with truncated lines? Are these autogenerated from something?